README.md

```diff
@@ -206,7 +214,50 @@ layeredWordCloud.build(1, wordFrequencies2);
 layeredWordCloud.writeToFile("output/layered_word_cloud.png");
 ```
 
-**Normalizers**
+Create a ParallelLayeredWordCloud using 4 distinct Rectangles.<br>
+Every Rectangle will be processed in a separate thread, thus minimizing build-time significantly
+
+```groovy
```

Comment:
I would prefer the examples be in Java along with the rest of the README. Sorry to nitpick.